src/pages/ErrorPage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Container, Image, VStack, Center } from '@chakra-ui/react';
+import './ErrorPage.css';
```

Comment:
let's make the `ErrorPage.css` file a `ErrorPage.module.css` so that styles don't accidentally leak into other components; everything is basically the same except:
- instead of `import './ErrorPage.css'` we import like `import styles from './ErrorPage.module.css'` 
- classes are defined by `className={styles.className}` instead of `className='className'`